convene-web/app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= page_title %></title>
+    <title><%= page_title %> [Convene Meeting]</title>
```

Comment:
```suggestion
    <title><%= page_title %> [Convene]</title>
```

I've submitted this as a [GitHub Suggestion](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request), which should make it (somewhat) easier for @driftcatcher to apply the requested change